base/shell/explorer/trayprop.cpp

```diff
@@ -197,8 +197,8 @@ class CStartMenuSettingsPage : public CPropertyPageImpl<CStartMenuSettingsPage>
         HWND hwndCustomizeModern = GetDlgItem(IDC_TASKBARPROP_STARTMENUCUST);
         HWND hwndStartBitmap = GetDlgItem(IDC_TASKBARPROP_STARTMENU_BITMAP);
         HWND hwndModernRadioBtn = GetDlgItem(IDC_TASKBARPROP_STARTMENU);
-        BOOL policyNoSimpleStartMenu = SHRestricted(REST_NOSTARTPANEL) ? TRUE : FALSE;
-        
+        BOOL policyNoSimpleStartMenu = SHRestricted(REST_NOSTARTPANEL) != 0;
```

Comment:
Returns 0 always. :disappointed:

Reply:
Yep, apparently Wine's implementation is buggy...
```
kd> da @@(regstr)
011cea9c  "Software\Microsoft\Windows\Curre"
011ceabc  "ntVersion\PoliciesExplorer"
```
Clearly a missing backslash